content/developer/reference/frontend/odoo_editor.rst

```diff
@@ -0,0 +1,120 @@
+
+.. highlight:: javascript
+
+.. default-domain:: js
+
+===========
+Odoo Editor
+===========
+
+Odoo Editor is Odoo's own rich text editor. It has replaced the Summernote
+library since Odoo 15.0. It can be found at
```

Comment:
We generally avoid mentioning past (or current) versions of Odoo in the documentation since it should be self-sufficient in the currently browsed version.
```suggestion
Odoo Editor is Odoo's own rich text editor.
It can be found at
```